code/_core/obj/item/deployable/_deployable.dm

```diff
@@ -8,8 +8,11 @@
 	item_count_max = 10
 	item_count_max_icon = 3
 
-/obj/item/deployable/proc/get_deploy_time(var/mob/caller)
-	return SECONDS_TO_DECISECONDS(3)
+/obj/item/deployable/proc/get_deploy_time(var/mob/caller, var/obj/item/deployable/D)
+	if(istype(D, /obj/item/deployable/mountable))
+		return SECONDS_TO_DECISECONDS(10)
+	else
+		return SECONDS_TO_DECISECONDS(3)
```

Comment:
This shouldn't be setup like this. You should make a new proc like

```
/obj/item/deployable/mountable/get_deploy_time(var/mob/caller, var/obj/item/deployable/D)
 return SECONDS_TO_DECISECONDS(10)
```

Reply:
That gives a duplicate definition, and would a need a separate on_click deploy for mountables or an if() in the deployable on_click instead. Do I separate mountable code from _deployable then?